example/server.rb

```diff
@@ -88,9 +92,36 @@
         'content-type' => 'text/plain',
       }, end_stream: false)
 
+      if options[:push]
+        push_streams = []
+
+        # send 10 promises
+        10.times do |i|
+          sleep 1
+          puts 'sending push'
+
+          head = { ':method' => 'GET',
+                   ':authority' => 'localhost',
+                   ':path' => "/other_resource/#{i}",
+                   'content-type' => 'text/plain' }
```

Comment:
content-type is a response header, it should move down.. or, we can just drop it.